packages/terra-core-docs/src/terra-dev-site/doc/form-select/example/search/Invalid.jsx

```diff
@@ -1,18 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
 import SearchSelect from 'terra-form-select/lib/SearchSelect';
 import classNames from 'classnames/bind';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
 import styles from '../FormSelectDocCommon.module.scss';
 
 const cx = classNames.bind(styles);
 
-const InvalidExample = () => (
-  <SearchSelect placeholder="Select a color" isInvalid className={cx('form-select')}>
-    <SearchSelect.Option value="blue" display="Blue" />
-    <SearchSelect.Option value="green" display="Green" />
-    <SearchSelect.Option value="purple" display="Purple" />
-    <SearchSelect.Option value="red" display="Red" />
-    <SearchSelect.Option value="violet" display="Violet" />
-  </SearchSelect>
-);
+const InvalidExample = () => {
+  const [isInvalid, setIsInvalid] = useState(true);
+  const [selectedValue, setSelectedValue] = useState('');
+  const [invalidText, setInvalidText] = useState(null);
+
+  const handleSelectChange = (value) => {
+    setIsInvalid(false);
+    setSelectedValue(value);
+  };
+
+  const handleFocus = () => {
+    setInvalidText('Please select a color');
+  };
+
+  return (
+    <>
+      <SearchSelect
+        placeholder="Select a color"
+        isInvalid={isInvalid}
```

Comment:
we should also add aria-invalid attribute for search-select-input control whenever `isInvalid` is set to true.

Reply:
It was already added in some other PR